packages/react-composites/src/composites/CallWithChatComposite/adapter/AzureCommunicationCallWithChatAdapter.ts

```diff
@@ -1044,6 +1161,17 @@ export const createAzureCommunicationCallWithChatAdapter = async ({
       'CallWithChat' as _TelemetryImplementationHint
     );
     callWithChatAdapter.setChatAdapterPromise(chatAdapterPromise);
+    /* @conditional-compile-remove(breakout-rooms) */
+    callWithChatAdapter.setCreateChatAdapterCallback((threadId: string) =>
```

Comment:
Should thie be inside `if (chatThreadAdapter.isCallInfoRequired())` or should it be after the else? (Does this not always need set?)

Reply:
I suppose it should always be set even though breakout rooms for now is only for Teams meetings. Updated.